README.md

```diff
@@ -3,19 +3,19 @@
 [![Travis Build Status](https://travis-ci.org/junaruga/rpm-py-installer.svg?branch=master)](https://travis-ci.org/junaruga/rpm-py-installer)
 [![Shippable Build Status](https://api.shippable.com/projects/5bb13c0997da11060049f4ad/badge?branch=master)](https://app.shippable.com/github/junaruga/rpm-py-installer/runs?branchName=master)
 
-`rpm-py-installer` is to enable [the RPM Python binding](https://github.com/rpm-software-management/rpm/tree/master/python) in any Python environment.
+`rpm-py-installer` is to enable [the RPM Python bindings](https://github.com/rpm-software-management/rpm/tree/master/python) in any Python environment.
 
-The environment is non-system Python, a source compiled Python, a Python on virtualenv, pyenv environment and etc.
+The environment can be a non-system Python, a source compiled Python, a Python in a virtualenv, pyenv environment, etc.
 
-It installs same version's Python binding with the system RPM safely.
+It installs the Python bindings matching the version of the system RPM in a safe manner.
 
 ## Getting started
 
-`rpm-py-installer` provides several ways to install RPM Python binding.
+`rpm-py-installer` provides several ways to install the RPM Python bindings.
 
-### Install by pip
+### Installation via pip
```

Comment:
I am okay to replace "by" with "via".
But for the "Installation", I am using "Install" style for following 3 ways to use same rhyme. How do you think?

```
README.md:16:### Installation via pip
README.md:52:### Install as a required dependency of your package
README.md:85:### Install the rpm Python bindings directly
```

Reply:
Changed it back to `Install`